packages/ui/src/styles/globals.css

```
@@ -213,27 +213,56 @@
    fold is the fixed strip revealed by the browser's own elastic overscroll,
    and rest stays at 0. --hero-rest is the park offset the scroll ranges key
    off, so both breakpoints resolve their own rest position. */
 @media (width < 64rem) and (hover: none) and (pointer: coarse) {
   :root:has(#hero-fold) {
-    --hero-fold: 140px;
-    --hero-rest: 140px;
+    /* Every pixel of park is pull the user eats before iOS arms pull-to-refresh
+       at scrollY 0, so: the status bar's ~60px, nothing spare. */
+    --hero-fold: 60px;
+    --hero-rest: var(--hero-fold);
     scroll-snap-type: y proximity;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)